source/features/update-pr-from-base-branch.tsx

```diff
@@ -49,16 +50,23 @@ async function handler({delegateTarget}: DelegateEvent): Promise<void> {
 async function addButton(position: Element): Promise<void> {
 	const {base, head} = getBranches();
 	const prInfo = await getPrInfo(base, head);
-	if (!prInfo) {
-		return;
-	}
 
-	if (prInfo.viewerCanEditFiles && prInfo.mergeable !== 'CONFLICTING') {
-		position.append(' ', (
-			<span className="status-meta d-inline-block rgh-update-pr-from-base-branch">
-				You can <button type="button" className="btn-link">update the base branch</button>.
-			</span>
-		));
+	if (prInfo.needsUpdate) {
+		if (pageDetect.isEnterprise()) {
+			position.append(' ', (
+				<span className="status-meta d-inline-block rgh-update-pr-from-base-branch">
+					You can <button type="button" className="btn-link">update the base branch</button>.
+				</span>
+			));
+		} else {
+			position.append(' ', (
+				<span className="status-meta d-inline-block">
+					{select('.head-ref')!.cloneNode(true)} is {pluralize(prInfo.headRef.compare.behindBy, '$$ commit', '$$ commits')} behind {select('.base-ref')!.cloneNode(true)}
```

Comment:
The head is available just a few words before this node, we shouldn't repeat it. If anything we can replace/reword it. 

Before doing further work, let's see how else we can word it or what information we actually want here.

Reply:
There's also the problem that this feature only appears when the native button doesn't (or at least that's the intention), so we probably shouldn't bundle this information with this feature. 

What if we just add it to the PR header next to the branches? Like:

> (Open) main <- feature (-5)

The number of "+" commits is already visible on the PR Commits tab. The number could show further information on in a tooltip. 

I'm not totally convinced by this however as the commit could be useful to be copied. I'm open to pushing a minimal version and then iterate on it as we start to rely on this piece of information. 

I'd also perhaps just append a simple version of this to both "Update branch" buttons (ours and the native one), and make it easier. Those areas at the bottom have plenty of space

Reply:
> I'd also perhaps just append a simple version of this to both "Update branch" buttons (ours and the native one), and make it easier. Those areas at the bottom have plenty of space

Lost you on this line

Reply:
> What if we just add it to the PR header next to the branches? Like:
> 
> > (Open) main <- feature (-5)

Like I as going to try to do this

Reply:
Should it be part of `clean-conversation-headers`

Reply:
> Lost you on this line

# Native

<img width="771" alt="Screen Shot 5" src="https://user-images.githubusercontent.com/1402241/199475406-e56e24f7-9eb1-4fa2-9957-856af46672b9.png">

# Ours

<img width="777" alt="Screen Shot 6" src="https://user-images.githubusercontent.com/1402241/199475444-a753eb39-f6c6-449e-8b83-f1ea2623087c.png">

# Change requested 

Show "base commit" near both of these. This feature isn't necessarily related to any other. It's not cleaning and it's not necessarily updating the branch.

Reply:
> > What if we just add it to the PR header next to the branches? Like:
> >
> > (Open) main <- feature (-5)
> Like I as going to try to do this, where are we putting the base commit link?

👇 

> The number could show further information in a tooltip.



With "further information" being the base commit. I followed that with a comment regarding the need to copy or see this value.

Reply:
In other words we should change it to?

![image](https://user-images.githubusercontent.com/16872793/200177161-5daa414e-b171-48e1-bed3-ab0f1d231c03.png)